stdlib/3/dbm/ndbm.pyi

```diff
@@ -0,0 +1,37 @@
+
+from typing import Generic, Union, Iterator, Optional, Type, TypeVar, overload, List
+from types import TracebackType
+
+_T = TypeVar('_T')
+_KeyType = Union[str, bytes]
+_ValueType = Union[str, bytes]
+
+class error(OSError): ...
+
+library: str = ...
+
+# Actual typename dbm, not exposed by the implementation
+class _dbm:
+
+    def close(self) -> None: ...
+    def __getitem__(self, item: _KeyType) -> _ValueType: ...
+    def __setitem__(self, key: _KeyType, value: _ValueType) -> None: ...
+    def __delitem__(self, key: _KeyType) -> None: ...
+    def __iter__(self) -> Iterator[_KeyType]: ...
+    def __len__(self) -> int: ...
+    def __del__(self) -> None: ...
+    def __enter__(self) -> _dbm: ...
+    def __exit__(self, exc_type: Optional[Type[BaseException]], exc_val: Optional[BaseException], exc_tb: Optional[TracebackType]) -> None: ...
+
+    @overload
+    def get(self, k: _KeyType) -> Optional[_ValueType]: ...
+    @overload
+    def get(self, k: _KeyType, default: Union[_ValueType, _T]) -> Union[_ValueType, _T]: ...
+    def keys(self) -> List[_KeyType]: ...
```

Comment:
I think this is just `List[bytes]` (and the same for most other return types).

Reply:
Fixed all the return types across all 3 modules, I think